### PR TITLE
[INT-889] Removed backslashes when using quotes

### PR DESCRIPTION
**Pull request by PolepalliGayathri**
Added stripslashes() before using the sanitised field name should remove the backslashes.

### CODE REVIEW

File: widget.subscribe.php [hunk 1]
```diff
@@ -342,7 +342,7 @@ function add_subscriber() {
 
 				if ( ! empty( $customfields[ $field_key ]['sailthru_customfield_name'] ) ) {
 					$name_stripped = preg_replace( '/[^\da-z]/i', '_', $customfields[ $field_key ]['sailthru_customfield_name'] );
-
+                    $name_stripped = stripslashes($name_stripped);
```
Comment (yogeshwarpst) [inline widget.subscribe.php:345]: correct the line formatting

File: widget.subscribe.php [hunk 2]
```diff
@@ -353,7 +353,7 @@ function add_subscriber() {
 							}
 						} else {
 							$var_name          = str_replace( 'custom_', '', $name_stripped );
-							$vars[ $var_name ] = sanitize_text_field( $_POST[ 'custom_' . $name_stripped ] );
+                            $vars[$var_name] = stripslashes(sanitize_text_field($_POST['custom_' . $name_stripped]));
```
Comment (yogeshwarpst) [inline widget.subscribe.php:356]: correct the line formatting

File: widget.subscribe.php [hunk 3]
```diff
@@ -353,7 +353,7 @@ function add_subscriber() {
 							}
 						} else {
 							$var_name          = str_replace( 'custom_', '', $name_stripped );
-							$vars[ $var_name ] = sanitize_text_field( $_POST[ 'custom_' . $name_stripped ] );
+                                                        $vars[$var_name] = stripslashes(sanitize_text_field($_POST['custom_' . $name_stripped]));
```
Comment (yogeshwarpst) [inline widget.subscribe.php:356]: lets also follow spacing convention as per the current line for e.g. [ $var_name ] vs [$var_name]

Comment (PolepalliGayathri) [reply widget.subscribe.php]: Changes made

**Review by yogeshwarpst — APPROVED** [review]
Looks fine to me

**Pull request merged**